modules/distr-tracing-tempo-config-multitenancy.adoc

```diff
@@ -159,9 +159,21 @@ spec:
             authenticator: bearertokenauth
           headers:
             X-Scope-OrgID: "dev"
+        otlphttp/dev: # <2>
+          endpoint: tempo-simplest-gateway.tempo.svc.cluster.local:8080
+          tls:
+            insecure: false
+            ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+          auth:
+            authenticator: bearertokenauth
+          headers:
+            X-Scope-OrgID: "dev"
       service:
         extensions: [bearertokenauth]
         pipelines:
           traces:
-            exporters: [otlp/dev]
+            exporters: [otlp/dev] # <3>
 ----
+<1> OTLP gRPC Exporter.
+<2> OTLP HTTP Exporter.
+<2> Enables the OTLP HTTP Exporter and disables the OTLP gRPC Exporter.
```

Comment:
@pavolloffay, :slightly_smiling_face: for when you are back from your vacation:
This is just a placeholder sentence.
I don't understand what has to happen here.
Could you please explain in some other words your original text below?
"The OTLP HTTP exporter. It needs to be specified in `service.pipelines.traces.exporters` to enable it. If it is enabled disable OTLP gRPC (`otlp/dev`)."